integration_test/helper.js

```diff
@@ -50,6 +51,58 @@ function Helper () {
     await new Promise(resolve => setTimeout(resolve, 1000))
     // await this.syncService.restart()
   }
+
+  this.cleanMemeolistDatabase = async (t) => {
+    t.log('Going to prepare memeolist database for the integration tests')
+
+    const {Client} = pg
+
+    const memeoListDbHost = process.env.MEMEOLIST_DB_HOST || '127.0.0.1'
+    const memeoListDbPort = process.env.MEMEOLIST_DB_PORT || '15432'
+
+    const client = new Client({
+      user: 'postgresql',
+      password: 'postgres',
+      database: 'memeolist_db',
+      host: memeoListDbHost,
+      port: memeoListDbPort
+    })
+
+    try {
+      await client.connect()
+      await client.query('SELECT 1')
+    } catch (err) {
+      t.log('Unable to connect memeolist database for preparing it for the integration tests')
+      throw err
+    }
+
+    try {
```

Comment:
IHMO it should be encapsulated in function prepareMemoList().
Also, in the top of the func it could have a comment gerated following the convention(it is done automatically by IDEA when you enter with `/*` ) as: 

```javascript
/******
* It will delete the tables if exists and create .... bla bla bla
* The database language here is SQL. 
*params ...
/
```